lib/model/request/submit_survey_answer_request.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part 'submit_survey_answer_request.g.dart';
+
+@JsonSerializable()
+class SubmitSurveyAnswerRequest {
```

Comment:
Can we name it `AnswerRequest` like we have `AnswerResponse` that is quite similar?

Reply:
Updated in [55e6acc](https://github.com/nimblehq/flutter-ic-lydia-ryan/pull/109/commits/55e6acc61c4bfdf9b96b54abeaaf3431243af4fa)